src/functions/removeCooldown.js

```diff
@@ -0,0 +1,21 @@
+module.exports = async (d) => {
+    const data = d.util.aoiFunc(d);
+    if (data.err) return d.error(data.err);
+    const data = d.util.aoiFunc(d);
+    const [command, type, ...id] = data.inside.splits;
+    const types = {
+        globalUser: id[0] || d.message.author.id,
+        user: `${id[0] || d.message.author.id}_${id[1] || d.message.guild.id || "dm"}`,
+        guild: id[0] || d.message.guild.id,
+        channel: id[0] || d.message.channel.id
+    };
+    if (!types[type]) return d.aoiError.fnError(d, "custom", { inside: data.inside }, "Invalid Type In");
+    if (await d.client.db.get(type == "guild" ? d.client.db.tables[0] : "__aoijs_vars__", "cooldown", command + "_" + types[type])) {
```

Comment:
Why?

Cooldowns are always stored into the internal `__aoijs_vars__`.

Reply:
You can check the `guildCooldown.js`, it seems to store the CD in the first db table instead of `__aoijs_vars__`.

Reply:
That's indeed unintended behaviour, which will be fixed in the next version. Therefore please change this to the `__aoijs_vars__` table.

Reply:
I see. I have committed a change. Please check it if it's okay.